From 8278e44557f93179f15f1a2cf44a570979a7024a Mon Sep 17 00:00:00 2001
From: Jian-Ming Wang <jian.ming.wang@mitaccomputing.com>
Date: Tue, 29 Jul 2025 08:21:35 +0000
Subject: [PATCH 1/1] Change the device name to follow the driver design.
 However, the libmctp-intel might be replaced by the driver mctp-pcie-vdm
 comming from Linux 6.15.x. So far, we just make it usable only.

Upstream-Status: Inappropriate [No upstream maintenance]
---
 astpcie.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/astpcie.h b/astpcie.h
index ff40d59..427ebfd 100644
--- a/astpcie.h
+++ b/astpcie.h
@@ -24,7 +24,7 @@ struct mctp_binding_astpcie {
 #define ASTPCIE_PACKET_SIZE(x) (ASPEED_MCTP_PCIE_VDM_HDR_SIZE + x)
 
 /* driver device file */
-#define AST_DRV_FILE "/dev/aspeed-mctp"
+#define AST_DRV_FILE "/dev/aspeed-mctp0"
 
 struct mctp_pcie_hdr {
 	uint8_t fmt_type;
-- 
2.45.2

